i3_agenda/cache_utils.py

```diff
@@ -0,0 +1,45 @@
+
+from config import CONF_DIR
+from typing import Optional, List
+
+import os.path
+import time
+import json
+
+from event import Event, EventEncoder
+
+CACHE_PATH = f"{CONF_DIR}/i3agenda_cache.txt"
+
+
+def load_cache(cachettl: int) -> Optional[List[Event]]:
+    if not os.path.exists(CACHE_PATH):
+        return None
+
+    if time.time() - os.path.getmtime(CACHE_PATH) > cachettl * 60:
+        return None
+
+    events = []
+
+    try:
+        with open(CACHE_PATH, "r") as f:
+            raw = json.loads(f.read())
```

Comment:
this block (in the with) cold be separated into another function

Reply:
Done